src/main/resources/com/google/api/codegen/java/fragment.snip

```diff
@@ -12,7 +12,7 @@
     /*
      * PRE-REQUISITES:
      * ---------------
-     * 1. If not already done, enable the {@ServiceName} API and check quota for your project at
+     * 1. If not already done, enable the {@ServiceName} API and check the quota for your project at
```

Comment:
This line isn't wrapped like the other languages. If the purpose of these changes is in part to not make users have to horizontally scroll, it may be worthwhile to wrap this line at 80 too even if with the API name it's likely to be under 100. And we should likely keep this in mind when we invoke the code formatter as well.

(Also: it's weird that in Java you have `{@ServiceName} API` but in other languages it's simply `{@serviceTitle}`. Wish we could be consistent.)